source/mongodump/mongodump-behavior.txt

```diff
@@ -209,6 +209,14 @@ following limitations apply:
   - :option:`--dumpDbUsersAndRoles`
   - :option:`--oplog <mongodump --oplog>`
 
+.. note:: Rollback on the target cluster during a restore or import
+   
+   If you are restoring or importing data to a cluster that undergoes a
```

Comment:
[Succinctness/Voice]: If the cluster undergoes a rollback during the restore process,

Reply:
Fixed!